components/metadata/rdm_parameters/rdm_pagination/rdm_pagination.py

```diff
@@ -53,7 +53,7 @@ def main():
 
     page_traits = MetadataPaginationTraits(a_page_size=args.page_limit, a_start=args.start)
     more_data = True
-    while True:
+    while more_data:
         metadata_list = query_metadata_by_domain_view(a_server_config=server, a_site_id=args.site_id, a_domain=args.rdm_domain, a_view=args.rdm_view, a_headers=headers, a_params=page_traits.params())
         more_data = page_traits.more_data(metadata_list)
         logging.info ("Found {} items {}".format(len(metadata_list["items"]), "({})".format("unpaginated" if page_traits.page_number() == 1 else "last page") if not more_data else "(page {})".format(page_traits.page_number())))
```

Comment:
How about we lose the inline "if"s and write 
```python
            num_items = len(data_page["items"]
            if more_data:
                logging.info("Found {} items (page {})".format(num_items, a_page_traits.page_number()))
            elif a_page_traits.page_number() == 1:
                logging.info("Found {} items (unpaginated)".format(num_items))
            else:
                logging.info("Found {} items (last page)".format(num_items))
```

Reply:
It was challenging to read wasn't it. Have split the conditions as suggested.